doc/nrf/ug_thingy91.rst

```diff
@@ -171,50 +171,28 @@ Building and programming using VS Code extension
 Complete the following steps to build |NCS| projects with |VSC|:
 
 1. Start |VSC|.
-
    If you installed the |NCS| using the :ref:`gs_app_tcm`, you can click the :guilabel:`Open VS Code` button next to the version you installed.
-
-#. Click :guilabel:`Create new application from sample` from the :guilabel:`Welcome` panel.
-
+#. In the Welcome panel or on the Welcome page, click :guilabel:`Add an existing application`.
    The :guilabel:`New Application` screen opens.
```

Comment:
This doesn't appear when using this method.

Reply:
Wille is correct. Instead a browser window opens and allows you to navigate your directories to find existing applications.